<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Libreria/background.pdf&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>361</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 166, 214, 210) ;
border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="UsernameField">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>200</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18pt; 
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 16pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);</string>
     </property>
     <property name="placeholderText">
      <string notr="true">Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordField">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>300</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18pt; 
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 16pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>413</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:24pt;
background-color: rgb(27, 25, 39);
color:rgb(255, 255, 255);
font: 22pt &quot;Source Code Pro for Powerline&quot;;
border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(27, 25, 39,220);
font: 18pt;
font: 22pt &quot;Mukta Mahee&quot;;
border:null;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;Mukta Mahee&quot;;
background:transparent;
color: rgb(27, 25, 39,220);
border:null;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border:null;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="User.qrc">:/icon/profile.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>384</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="remember_box">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border:none;
</string>
     </property>
     <property name="text">
      <string>Mantieni l'accesso</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(27,25,39);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/Logo/Babelib-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(27,25,39);
border-radius: 40px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>-150</x>
      <y>0</y>
      <width>931</width>
      <height>731</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="wa.qrc">:/waa/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>frame</zorder>
   <zorder>toolButton</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
  <include location="wa.qrc"/>
  <include location="Sfondo.qrc"/>
  <include location="Icon.qrc"/>
  <include location="User.qrc"/>
 </resources>
 <connections/>
</ui>
